modules/ROOT/pages/server-group-about.adoc

```diff
@@ -3,9 +3,12 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-A server group is a set of servers that act as a single deployment target, in which instances of the applications are completely isolated from each other.
+A server group is a set of servers that act as a single deployment target for applications, which provides the advantage of not having to deploy an application to each server individually. Applications are deployed to each server in the server group through a single deployment and run in isolation from the application instances running on the other servers. 
```

Comment:
This is a great place to add a recommended approach for Disaster Recovery to ensure Customers Run their Servers in Server Groups.  If not already described in this page, can we add a statement for this please? Thanks!